lib/Models/CatalogItem.js

```diff
@@ -212,18 +200,27 @@ var CatalogItem = function(terria) {
      * @memberOf CatalogItem.prototype
      */
     knockout.defineProperty(this, 'dataUrlType', {
-        get : function() {
+        get: function() {
             if (defined(this._dataUrlType)) {
                 return this._dataUrlType;
             } else {
                 return 'direct';
             }
         },
-        set : function(value) {
+        set: function(value) {
             this._dataUrlType = value;
         }
     });
 
+    knockout.defineProperty(this, 'legendUrls', {
```

Comment:
Not much reason to use `knockout.defineProperty` for a property like this that just delegates to a private field.  Just make a public field instead.